cli/commands/benchmark-web-vitals.mjs

```diff
@@ -0,0 +1,220 @@
+/**
+ * CLI command to benchmark several URLs for Core Web Vitals and other key metrics.
+ *
+ * WPP Research, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import puppeteer from 'puppeteer';
+import round from 'lodash-es/round.js';
+
+/**
+ * Internal dependencies
+ */
+import { getURLs } from '../lib/cli/args.mjs';
+import {
+	log,
+	formats,
+	table,
+	isValidTableFormat,
+	OUTPUT_FORMAT_TABLE,
+} from '../lib/cli/logger.mjs';
+import { calcMedian } from '../lib/util/math.mjs';
+
+export const options = [
+	{
+		argname: '-u, --url <url>',
+		description: 'An URL to run benchmark tests for',
```

Comment:
```suggestion
		description: 'A URL to run benchmark tests for',
```

@felixarntz I've already approved, and this is not a deal breaker, but `A` would be more grammatically correct than `An` in this instance. More of a nitpick than anything else.

Reply:
@10upsimon Thanks, fixed! A similar mistake was present in the `benchmark-server-timing.mjs` file too so I fixed it in both places.